test/Quaternion.jl

```diff
@@ -0,0 +1,442 @@
+using Quaternions
+using LinearAlgebra
+using Random
+using Test
```

Comment:
Do we need these `using` here? I thought this can be moved to `runtests.jl`.

Reply:
Ah, I was always commenting out the `include` in `runtests.jl` for that. :smile: